hyracks/hyracks-dataflow-std/pom.xml

```diff
@@ -64,5 +64,10 @@
           <version>0.2.17-SNAPSHOT</version>
           <scope>test</scope>
       </dependency>
+      <dependency>
```

Comment:
we shall not add the dependency to dataflow-std. it will end with an dependency circle.